docs/source/technical_tutorials/access/direct_access.ipynb

```diff
@@ -0,0 +1,1577 @@
+{
```

Comment:
<p>Should we also demo rasterio direct usage in one example?</p><p>And should the rioxarray actually be through Xarray directly?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/344/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<ol><li>is done below</li><li>is correct based on rioxarray docs</li></ol>